agriculture_pricing/README.md

```diff
@@ -1,3 +1,154 @@
 # Agriculture Pricing Example
 
-Example of implementation of agriculture pricing defined [here](https://github.com/Gurobi/modeling-examples/tree/master/agricultural_pricing).
\ No newline at end of file
+Example of implementation of agriculture pricing adapted from [here](https://github.com/Gurobi/modeling-examples/tree/master/agricultural_pricing).
+
+## Data View
+
+Columns marked with __*__ are index columns.
+
+#### `capacity.csv`
+
+| `component`* | `capacity` |
+| --- | --- |
+| __Defines__ set <u>Components</u> ($C$) | __Defines__ param <U>Last Year's Demand |
+
+> Missing index values allowed? : N/A
+
+#### `composition.csv`
+
+| `dairy`* | `component`* | `composition` |
+| --- | --- | -- |
+| Elements __belong__ to <u>Dairy ($D$)</u> | Elements __belong__ to set <u>Components ($C$)</u> | __Defines__ param <u>Composition</u> |
+
+> Missing index values allowed? : No
+
+#### `constants.csv`
+
+| `price_index` |
+| --- |
+| __Defines__ param <u>Price Index</u> |
+
+> Missing index values allowed? : N/A
+
+#### elasticity.csv
+
+| `dairy_1`* | `dairy_2`* | `elasticity` |
+| --- | --- | -- |
+| Elements __belong__ to <u>Dairy ($D$)</u> | Elements __belong__ to <u>Dairy ($D$)</u> | __Defines__ param <u>Elasticity</u> |
+
+> Missing index values allowed? : Yes
+
+#### `market.csv`
+
+| `dairy`* | `demand` | `price` |
+| --- | --- | -- |
+| __Defines__ set <u>Dairy</u> ($D$) | __Defines__ param <u>Last Year's Demand | __Defines__ param <u>Last year's price |
+
+> Missing index values allowed? : N/A
+
+## Model View
+
+
+#### Dairy (Set), $\text{D}$
+
+> Source: From `index columns` of `market.csv`.
+
+#### Components (Set), $\text{C}$
+
+> Source: From `index columns` in `capacity.csv`.
+
+#### Capacity (Param), $\text{capacity}_c$
+
+$$0 \le \text{capacity}_{c} \quad \forall c \in C$$
+
+> Description: Yearly availability of component $c$ (1000 tons).
+
+> Source: From column `capacity` in `capacity.csv`.
+
+#### Composition (Param), $\text{qtyper}_{c,d}$
+
+$$0 \le \text{qtyper}_{c,d}\le 1 \quad \forall c\in C, d\in D$$
```

Comment:
the variable name is inconsistent with the solve.py file. 
I think we should have two versions of the agricultural_pricing model. One with the generalized equation (yours) and the original version, because they have different latex -> mathjson -> code problems (e.g. \sum_{'milk'} and \sum_{d1 != d2}.

Reply:
I haven't tried to make the variable names consistent since afaik we haven't decided on a method for that (yet)? Do you want the code variable name to come from the title (e.g. Composition) or from the latex variable name $qtyper$?

On your second point, I agree if we have time it would be good to add the second version to this example database (the one with $\sum_{milk}$).

Reply:
Oh, I meant consistent with the solve.py file var names. You use composition instead of qtyper, last_year_price instead of price, price instead of p, etc...

Reply:
yes because instead of using the variable names in the latex (E.g. qtyper) I used the variable names in the text header. Those are consistent now with solve.py